examples/chat/src/main.rs

```diff
@@ -69,6 +69,9 @@ use std::str::FromStr;
 use std::sync::{Arc, Mutex};
 use tracing::info;
 
+// Unique namespace to avoid cryptographic hash collisions.
```

Comment:
nit: `Unique namespace to avoid message replay attacks.`

Reply:
I have some words around this here: https://github.com/commonwarexyz/monorepo/blob/793d38e48329156b739e83bded26a8f6a1417ca0/cryptography/src/lib.rs#L57-L60